README.md

```diff
@@ -1,229 +1,144 @@
 <a href="http://wso2.com/products/identity-server/">
-<img src="http://b.content.wso2.com/sites/all/common/images/product-logos/identity-server.svg"
-     srcset="http://b.content.wso2.com/sites/all/common/images/product-logos/identity-server.svg@2x.png 2x"
-     alt="WSO2 Identity Server" />
+<img src="https://wso2.cachefly.net/wso2/sites/all/image_resources/wso2-branding-logos/wso2-logo-orange.png" alt="WSO2 logo" width=30% height=30% />
```

Comment:
Is it expected to replace the identity server logo with the WSO2 logo?

Reply:
@sagara-gunathunga

Reply:
This is an intentional change  to have WSO2 brand name across all of the  products in a consistent manner.  Please refer to the references given in the description.  We are adopting the same template across all the products, please refer to how this has been implemented in other products.  

https://github.com/isamauny-wso2/product-apim/blob/readme_update_q424/README.md?plain=1 
https://github.com/wso2/product-micro-integrator/pull/4145/files
https://github.com/wso2/product-streaming-integrator/pull/330/files
https://github.com/wso2/product-ballerina-integrator/pull/258/files